custom-css-and-js-in-sphinx.rst

```diff
@@ -0,0 +1,94 @@
+.. post:: July 10, 2019
+   :tags: sphinx
+   :author: David
+   :location: SAN
+
+.. meta::
+   :description lang=en:
+       Customize your documentation by adding CSS stylesheets or JavaScript files
+       to change your docs' look and feel
+
+Adding Custom CSS or JavaScript to Sphinx Documentation
+=======================================================
+
+In the Read the Docs documentation, we have a number of
+`how-to guides`_
+to help people solve specific problems with Sphinx and Read the Docs.
+By far our most popular guide is on
+`adding custom CSS and JavaScript to Sphinx`_.
+
+In some older versions of Sphinx, this process was a little more challenging
+and it wasn't as easy to figure out how to do it from the Sphinx docs.
+Sphinx 1.8 really streamlined this process especially for the simple cases.
+
+.. _how-to guides: https://docs.readthedocs.io/page/guides/index.html
+.. _adding custom CSS and JavaScript to Sphinx: https://docs.readthedocs.io/page/guides/adding-custom-css.html
+
+
+Adding additional stylesheets or scripts
+----------------------------------------
+
+If your custom stylesheet is ``_static/css/custom.css``,
+you can add that CSS file to the documentation using the
+Sphinx option `html_css_files`_::
+
+    ## conf.py
```

Comment:
nit: it would be good to use `.. code-block:: python` directive so we get the highlight here and in the following one as well.

Reply:
The blog highlights Python by default. It's better to be explicit though so I'll update it.